app/src/main/res/values-night/colors.xml

```diff
@@ -3,70 +3,147 @@
-    <color name="settings_whats_new">@color/accentBright</color>
+    <color name="colorCursor">#ccffffff</color>
+    <color name="colorOverlay">#cc000000</color>
 
-    <color name="trackingProtectionSheetBackground">#1D1133</color>
-    <color name="primaryTextColor">#FBFBFE</color>
+    <color name="toolbarUrlBackground">@color/photonInk05</color>
 
-    <color name="colorPrimary">@color/photonInk60</color>
-    <color name="colorSecondary">@color/photonInk05</color>
-    <color name="menuBackground">@color/colorSecondary</color>
-    <color name="toolbarUrlDarkBackground">@color/photonInk05</color>
-    <color name="toolbarUrlLightBackground">#FBFBFE</color>
+    <color name="divider">@color/photonInk10</color>
+    <color name="accentBright">@color/photonPink40</color>
+    <color name="accent">@color/photonPink70</color>
+    <color name="destructive">#FF6A75</color>
+
+    <!-- Logo and Splash -->
     <color name="headerGradientStart">#2B2154</color>
     <color name="headerGradientMiddle">#281D4C</color>
     <color name="headerGradientEnd">#2A2051</color>
-    <color name="divider">#342F6D</color>
-    <color name="accentBright">#FF4AA2</color>
-    <color name="accent">#C60084</color>
-    <color name="destructive">#FF6A75</color>
-    <color name="primaryText">@color/photonLightGrey05</color>
-    <color name="secondaryText">@color/photonLightGrey50</color>
+
+    <color name="homeGradient1">#FF1E1236</color>
+    <color name="homeGradient2">#FF332866</color>
+    <color name="homeGradient3">#FF4D3A83</color>
+    <color name="homeGradient4">#FF3C286A</color>
+    <color name="homeGradient5">#FF21163E</color>
+    <color name="homeGradient6">#FF1D1133</color>
+
+    <!-- Progress Bar -->
+    <color name="colorProgressGradientStart">#FFFF1AD9</color>
+    <color name="colorProgressGradientCenter">#FFFF1AD9</color>
+    <color name="colorProgressGradientEnd">#FFFF1AD9</color>
+
+    <color name="colorErase">@color/photonMagenta70</color>
+    <color name="colorErasePressed">@color/photonMagenta80</color>
+
+    <color name="colorDotHighlight">@color/photonBlue40</color>
+    <color name="colorCrashAccent">@color/photonBlue40</color>
+
+    <!-- Toolbar -->
+    <color name="toolbar_color">@color/colorPrimary</color>
+
+    <color name="images_removal_warning_background_color">#353852</color>
+    <color name="colorPoweredBy">@color/photonLightGrey05</color>
+    <color name="menuBackground">@color/colorSecondary</color>
+
+    <!-- Snackbar-->
+    <color name="snackbarBackground">#FF414146</color>
+    <color name="snackbarTextColor">#ffffffff</color>
+    <color name="snackbarActionText">@color/photonBlue50</color>
+
+    <!-- Search Suggestions -->
+    <color name="searchSuggestionsBackground">@color/photonInk80</color>
+    <color name="searchSuggestionPromptBackground">@color/photonInk80</color>
+    <color name="searchSuggestionPromptTitleColor">@color/photonLightGrey05</color>
```

Comment:
I only came around because I was talking to Mugurell about design systems and colors since I am providing some engineering context for Crystal who is working on the design system. 

Can we match all these instances of `photonLightGrey05` to `colorPrimary`? It seems to me overall, we want to map to these "Primary", "Secondary" variables in the long run instead of the photon color directly. Since UX can easily make the decision to change "Primary" from say "photonLightGrey05" to "photonLightGrey20", that way we will just propagate the changes all around.

Reply:
This is not intended to block the PR or anything. Just more design system thinking from my side and trying to rationalize how we should go about this.

Reply:
Feel free to do this as a followup.

Reply:
I do not have designs for the light theme of this, so I cannot know if the corresponding light theme color would still be `colorPrimary`. But yes, as a rule, I tried merging colors to refer to a small number of colors.